cms/static/js/views/import.js

```diff
@@ -97,8 +97,8 @@ define(
             $.cookie(COOKIE_NAME, JSON.stringify({
                 file: file,
                 date: moment().valueOf(),
-                completed: completed || false
-            }));
+                completed: completed || false,
+            }), {path: window.location.pathname});
```

Comment:
do we also want to limit how long these cookies are valid for? Maybe 14 days instead of a year?